qcengine/procedures/geometric.py

```diff
@@ -24,24 +24,24 @@ def found(self, raise_error: bool = False) -> bool:
     def build_input_model(self, data: Union[Dict[str, Any], "OptimizationInput"]) -> "OptimizationInput":
         return self._build_model(data, OptimizationInput)
 
-    def compute(self, input_data: "OptimizationInput", config: "TaskConfig") -> "OptimizationResult":
```

Comment:
Unsure why this file was changed? Please consider reverting.

Reply:
I believe that was part of #166. I assumed the purpose was to unify the two procedures. Happy to revert if you'd like

Reply:
Yeah, pretty sure that's part of my campaign to help readability by having the the var names for the object be `_model` and the dict be `_data`.

Reply:
Ok, good to leave it.